web/public/templates/profiles/programming-languages-list.dust

```diff
@@ -1,8 +1,6 @@
 <div class="profile-section cd-programming-languages">
   <h3 class="cd-section-header"><a id="#prog"></a><a href="#prog">{@i18n key="Programming Languages"/}</a></h3>
-  <div>
-    <ul ng-repeat="language in profile.widget.programmingLanguages" class="list-unstyled">
-      <li class="col-md-6"> <span class="badge">{{ language }}</span></li>
-    </ul>
+  <div class="col-sm-2" ng-repeat="language in programmingLanguagesImageUrl">
```

Comment:
please define a size for xs display, i think keeping the same ratio over a small display is fine, a prog language isn't the primary info of a profile